redis/src/main/scala/zio/redis/ResultBuilder.scala

```diff
@@ -33,18 +33,18 @@ object ResultBuilder {
   final abstract class NeedsReturnType
 
   trait ResultBuilder1[+F[_]] extends ResultBuilder {
-    def returning[R: Schema]: ZIO[Redis, RedisError, F[R]]
+    def returning[R: Schema]: ZIO[RedisEnv, RedisError, F[R]]
```

Comment:
Please refrain from making non-related changes. Renaming `Redis` to `RedisEnv` does not bring any value other than increasing the diff size and introducing a confusing name, so please restore it.

Reply:
I added when we had zio 1, and i have to propagate everywhere logger.
No it useless - return Redis)